src/07-even-more-loops.js

```diff
@@ -96,7 +150,21 @@ function evenArray(array) {}
  *  findSmallest([ 0, 11, -2, 5 ]);
  *  //> -2
  */
-function findSmallest(array) {}
+function findSmallest(array) {
+  //infinity is the biggest number, we are looking for the smallest
+  let smallest = Infinity;
```

Comment:
This is an interesting concept here but setting smallest to ` Infinity ` is not needed.  Since we are iterating over an array we can set our smallest to the first element in the array and then move forward with our comparison.